packages/firebase_remote_config/lib/src/remote_config_fetch_throttled_exception.dart

```diff
@@ -0,0 +1,18 @@
+part of firebase_remote_config;
+
+class FetchThrottledException implements Exception {
```

Comment:
I would put some dartdoc here, e.g. "Exception thrown when the fetch() operation cannot be completed successfully, due to throttling."

Reply:
Done